runbot_send_email/models/runbot_build.py

```diff
@@ -0,0 +1,154 @@
+# coding: utf-8
+# © 2016 Vauxoo
+#   Coded by: lescobar@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+from openerp import api, fields, models, _
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotBuild(models.Model):
+    _name = "runbot.build"
+    _inherit = ['runbot.build', 'mail.thread']
+
+    repo_name = fields.Char('Repo Name', compute='_repo_name')
+    branch_name = fields.Char('Branch Name or PR#', compute='_branch_name')
+    subject_email = fields.Char('Subject Email', compute='_subject_email')
+    webaccess_link = fields.Char('Web Access Link', compute='_webaccess_link')
+    logplain_link = fields.Char('Log Link', compute='_logplain_link')
+    log_link = fields.Char('Log Link', compute='_log_link')
+    ssh_link = fields.Char('SSH Link', compute='_ssh_link')
+    doc_link = fields.Char('Documentation Link', compute='_doc_link')
+    dockerdoc_link = fields.Char('Docker Link', compute='_dockerdoc_link')
+    configfile_link = fields.Char('Config File Link',
+                                  compute='_configfile_link')
+    shareissue_link = fields.Char('Share Issue Link',
```

Comment:
And in odoo 9.0 if you call a variable with the same name of the label, labe is optional, ORM will set the word for you correctly